### DISCUSSION TITLE
Support for Central Europe regions?

### DISCUSSION THREAD

**Discussion by ivan-kleshnin** [vercel/vercel#5160] [Ideas] [answered]
AWS has a datacenter in **Frankfurt**: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html 

The list of supported Vercel regions hovewer does not have it: https://vercel.com/docs/edge-network/regions

Are there any plans to extend Vercel regions to include some Central Europe locations?  

AWS also has an Edge Network node in **Berlin** according to https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services. Would be great to have as well... if CDN regions can be different from Lambda regions at Vercel.

**Comment by styfle** [comment, marked as answer]
Hi @ivan-kleshnin 

That's a great idea!

We are always [making improvements](https://vercel.com/blog/new-edge-dev-infrastructure) to our global network and will likely add more regions in the future.

We don't have an ETA for new regions at this time, thanks!
